### DISCUSSION TITLE
哪一个是解析的结果

### DISCUSSION THREAD

**Discussion by hx49720** [magicpdf/Magic-PDF#300] [Q&A]
解析后形成目录如下，哪一个是解析的结果？
![微信截图_20240802112537](https://github.com/user-attachments/assets/224bc556-4161-4f33-8770-e217e65b6249)

尝试打开了1、申报表_model.json，但是内容和PDF的内容差距较远
[
![微信截图_20240802112633](https://github.com/user-attachments/assets/2a04c2fc-9600-4009-9112-cb1efc40cebd)
](url)

**Comment by myhloli** [comment]
md那个
